ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -57,6 +59,46 @@
            subscriber-id
            concept-id)))))
 
+(defn- subscription-duplicate-error
```

Comment:
This function contains one function call and it is called just once in the code base. We don't need this function. Just call `(errors/throw-service-error ...)` in `check-duplicate-subscription` is more readable.